engine/execution/computation/computer/computer.go

```diff
@@ -401,19 +400,31 @@ func (e *blockComputer) executeTransaction(
 	// of failed transaction invocation
 	err = collectionView.MergeView(txView)
 	if err != nil {
-		return fmt.Errorf("merging tx view to collection view failed: %w", err)
+		return fmt.Errorf("merging tx view to collection view failed for tx %v: %w",
```

Comment:
this path would have zero logs, does it make sense to still log something when we fail here?

Reply:
We do have log for this error case.

[The error will popup, and eventually logged with other context, including the block id.](https://github.com/onflow/flow-go/blob/0ffe937b9d0077f6fc4889d334d2dedd6a4b834b/engine/execution/ingestion/engine.go#L575)

I don't think it's necessary to log again.